### PR TITLE
[test](regression-test) fix case bug suites/export/test_array_export.groovy

### PR DESCRIPTION
**Pull request by shuke987**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by shuke987** [comment]
run buildall

File: regression-test/suites/export/test_array_export.groovy [hunk 1]
```diff
@@ -174,7 +174,7 @@ suite("test_array_export", "export") {
         result = sql """
             SELECT * FROM ${tableName} t ORDER BY k1 INTO OUTFILE "file://${outFile}/";
         """
-        url = result[0][3]
+        def url = result[0][3]
         urlHost = url.substring(8, url.indexOf("${outFile}"))
```
Comment (cambyzju) [inline regression-test/suites/export/test_array_export.groovy:178]: maybe other variables also have the same problem, such as `urlHost` ?

Comment (shuke987) [reply regression-test/suites/export/test_array_export.groovy:178]: urlHost is defined as local variable

**Review by hello-stephen — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**